apps/projection/src/app/ui/card/card.component.ts

```diff
@@ -21,14 +26,17 @@ export class CardComponent {
 
   constructor(
     private teacherStore: TeacherStore,
-    private studentStore: StudentStore
+    private studentStore: StudentStore,
+    private cityStore: CityStore
   ) {}
 
   addNewItem() {
     if (this.type === CardType.TEACHER) {
       this.teacherStore.addOne(randTeacher());
     } else if (this.type === CardType.STUDENT) {
       this.studentStore.addOne(randStudent());
+    } else if (this.type === CardType.CITY) {
+      this.cityStore.addOne(randomCity());
     }
```

Comment:
This should not be here. Each store should stay in its specific component. 
use @Output() to emit data to your parent